README.md

```diff
@@ -38,105 +35,179 @@ DesignGuide requires [Node.js](https://nodejs.org/en/) (includes npm).
+| `CHANGELOG.md`      | Collection of changes made to the project. Insert your changes here.
+| `jest.config.js`    | [jest][jest] configuration.
+| `jest.setup.js`     | Script file that will run before jest executes the tests, this includes [enzyme][enzyme] setup, polyfills and other useful snippets.
+| `webpack.config.js` | [Webpack][webpack] configuration.
 
 ## Copyright, license and credits
 
 Code and documentation © Swedbank Pay and contributors, released under the [MIT License](LICENSE).
 
-Inspired by [Bootstrap](http://getbootstrap.com/) and [Materialize](https://materializecss.com/).
+Inspired by [Bootstrap][bootstrap] and [Materialize][materialize].
+
+  [version-badge]:      https://img.shields.io/badge/Version-4.1.0-blue.svg
+  [version-tag]:        https://github.com/swedbankpay/design.swedbankpay.com/releases/tag/4.1.0
+  [appveyor-badge]:     https://ci.appveyor.com/api/projects/status/1dii19sqw1m7xtsn/branch/master?svg=true
+  [appveyor-status]:    https://ci.appveyor.com/project/PayEx/design-swedbankpay-com/branch/master
+  [appveyor]:           https://www.appveyor.com/
+  [codecov-badge]:      https://codecov.io/gh/swedbankpay/design.swedbankpay.com/branch/develop/graph/badge.svg
+  [codecov-status]:     https://codecov.io/gh/swedbankpay/design.swedbankpay.com
+  [dependabot-badge]:   https://api.dependabot.com/badges/status?host=github&repo=SwedbankPay/design.swedbankpay.com
+  [dependabot]:         https://dependabot.com
+  [dep-badge]:          https://david-dm.org/swedbankpay/design.swedbankpay.com/status.svg
+  [dep-status]:         https://david-dm.org/swedbankpay/design.swedbankpay.com
+  [devdep-badge]:       https://david-dm.org/swedbankpay/design.swedbankpay.com/dev-status.svg
+  [devdep-status]:      https://david-dm.org/swedbankpay/design.swedbankpay.com?type=dev
+  [og-image]:           https://repository-images.githubusercontent.com/191536903/a4653c80-e7ac-11e9-9d96-c5bf40f73d9c
+  [swpdg]:              https://design.swedbankpay.com
+  [nodejs]:             https://nodejs.org/en/
+  [slack]:              https://payex.slack.com/messages/C0L3W8B2S/
+  [react]:              https://reactjs.org/
+  [less]:               http://lesscss.org/
+  [webpack]:            https://webpack.js.org/
+  [semver]:             http://semver.org/
+  [issues]:             https://github.com/swedbankpay/design.swedbankpay.com/issues
+  [erikhallaraaker]:    https://github.com/erikhallaraaker
+  [hmosvoll]:           https://github.com/hmosvoll
+  [VikingTristan]:      https://github.com/VikingTristan
+  [babel]:              https://babeljs.io/
+  [eslint]:             https://eslint.org/
+  [sentry]:             https://sentry.io
+  [stylelint]:          https://stylelint.io/
+  [jest]:               https://facebook.github.io/jest/
+  [enzyme]:             http://airbnb.io/enzyme/
+  [bootstrap]:          http://getbootstrap.com/
+  [materialize]:        https://materializecss.com/
```

Comment:
Whoah, didn't know you could create variables like this in markdown. Pretty slick!